src/functions.js

```diff
@@ -109,16 +169,40 @@ function sales( cars ){
   //   'Ford': 20000,
   //   'Vauxhall': 15000
   // }
+  // var sale = {};
+  // var temp = 0;
+  // for (i=0; i < cars.length, i++){
+    
+  // }
+  var newObj = { 'Ford': 0, 'Land Rover': 0, 'Toyota': 0, 'Honda': 0 };
+  for (var i = 0; i < cars.length; i++) {
+    for (var key in cars[i]) {
+      if (cars[i][key] === 'Ford') newObj['Ford'] += cars[i]['price']
+      if (cars[i][key] === 'Land Rover') newObj['Land Rover'] += cars[i]['price']
+      if (cars[i][key] === 'Toyota') newObj['Toyota'] += cars[i]['price']
+      if (cars[i][key] === 'Honda') newObj['Honda'] += cars[i]['price']
+    }
+  }
+  return newObj;
 }
```

Comment:
Whilst this one works, give some thought to what would happen if the test array contained BMW, Mercedes and Fiat vehicles. Hard coding values into a function is rarely a good idea, as it limits their reuse, which is a functions' main purpose. I'd be happy to take you through a different (reusable) solution tomorrow.